src/Collisions/babylon.collisionCoordinator.ts

```diff
@@ -390,7 +392,8 @@ module BABYLON {
             // Check all meshes
             for (var index = 0; index < this._scene.meshes.length; index++) {
                 var mesh = this._scene.meshes[index];
-                if (mesh.isEnabled() && mesh.checkCollisions && mesh.subMeshes && mesh !== excludedMesh) {
+                if(!mesh._collisionGroup) mesh._collisionGroup = -1;

```

Comment:
Here we can have a local variable named collisionMask = (excludedMesh  ? excludedMesh._collisionMask : -1)